app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -146,6 +157,10 @@
     // latest configuration
     AaaConfig newCfg;
 
+    ScheduledExecutorService ses = Executors.newScheduledThreadPool(1);
```

Comment:
use better name for variable than ses. Just executor is also fine. Also, initialize it in activate() method, else a thread of this app will be running even when application is not active